### DISCUSSION TITLE
南墙跨域配置

### DISCUSSION THREAD

**Discussion by OnionTop** [Safe3/uusec-waf#39] [Q&A]
我们前后端分离，然后前后端域名是不同的项目，默认前端访问后端会出现跨域问题，所以要手动加一些跨域相关配置，允许自定义的请求头、方法一类的信息，但是不知道怎么配了，没找到配置的地方，哪位大佬帮忙解答，跪谢！！

**Comment by Safe3** [comment]
可以在/uuwaf/conf/uuwaf.conf中配置，然后执行systemctl restart uuwaf生效

**Comment by wen0750** [comment]
我正在尝试修改资料库配置以连接到自托管的资料库。 我已修改了 /uuwaf/conf/uuwaf.conf 和 /uuwaf/web/conf/conf.yaml 档案中的 host, port, user, password。 然而进行这些修改后frontend提示无法连接到资料库。 有成功的案例能分享一下吗? 跪谢！

**Reply by Safe3** [reply]
可以加官方群给你看看
